core/deployment/src/main/java/io/quarkus/deployment/dev/ClassLoaderCompiler.java

```diff
@@ -1,200 +0,0 @@
-package io.quarkus.deployment.dev;
-
-import java.io.Closeable;
-import java.io.File;
-import java.io.IOException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.net.URLDecoder;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.ArrayDeque;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.jar.Attributes;
-import java.util.jar.JarFile;
-import java.util.jar.Manifest;
-import java.util.regex.Pattern;
-
-import org.jboss.logging.Logger;
-
-import io.quarkus.bootstrap.app.CuratedApplication;
-import io.quarkus.bootstrap.model.AppDependency;
-
-/**
- * Class that handles compilation of source files
- *
- * @author Stuart Douglas
- */
-public class ClassLoaderCompiler implements Closeable {
```

Comment:
This was renamed as the old name is no longer relevant, it used to compile by reading the current ClassPath from the ClassLoader, and it has not done that for a very very long time.